docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -878,6 +878,51 @@ Consider the example below for reading incremental loading parameters from "conf
    ```
    `id_after` incrementally stores the latest `cursor_path` value for future pipeline runs.
 
+### Loading NULL values in the incremental cursor field
+
+When loading incrementally with a cursor field, each row is expected to contain a value at the cursor field that is not `None`.
+For example, the following source data will raise an error:
+```py
+data = [
+    {"id": 2, "updated_at": 2},
+    {"id": 1, "updated_at": 1},
+    {"id": 3, "updated_at": None},
+]
```

Comment:
Alternatively, we could give an example of a source crashing:

```py
@dlt.resource
def some_data(updated_at=dlt.sources.incremental("updated_at")):
    yield [
        {"id": 1, "created_at": 1, "updated_at": 1},
        {"id": 2, "created_at": 2, "updated_at": 2},
        {"id": 3, "created_at": 4, "updated_at": None},
    ]

list(some_data())
```

Reply:
I like the source example a bit more because it's closer to real life